setup.cfg

```diff
@@ -44,6 +44,8 @@ disallow_untyped_calls = True
 disallow_incomplete_defs = True
 disallow_untyped_defs = True
 disallow_untyped_decorators = True
+# FIXME: should be switch to True (default) after fixing implicit Optional in the codebase
```

Comment:
Should I create a ticket for this? There are not that many such errors (~40 places), but still too large to do in one run

Reply:
Is this as a result of the mypy upgrade? We already have a [ticket](https://rasahq.atlassian.net/browse/ATO-588) to resolve new issues surfaced by this upgrade, please add a comment there and update the title of the ticket.

Reply:
Yes, change of this default happened with 0.99 mypy release. As there are not that many warnings about this and changes are fairly trivial (most of the time, adding `Optional` is enough), I think it makes sense to upgrade mypy and eventually switch to defaults in the future, rather than use old mypy.

Will update the ticket.